ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -370,28 +370,31 @@ def test_ipa_healthcheck_fips_enabled(self):
 
         https://pagure.io/freeipa/issue/8951
         """
+        healthcheck_version = tasks.get_healthcheck_version(self.master)
+        if parse_version(healthcheck_version) <= parse_version("0.17"):
+            # Patch: https://github.com/freeipa/freeipa-healthcheck/pull/349
+            pytest.xfail("Patch is unavailable for freeipa-healtheck "
+                         "version 0.17 or less")
+
```

Comment:
With this xfail the test is always skipped. That is not the intended behavior, it should be marked xfail if version < 0.17 (strictly <, not <=) AND os = rhel 10. For fedora the utility "fips_mode_setup" is still shipped and the test must be run.

Reply:
Oh alright, thanks for letting me know, I've expected that a new version of healthcheck would have to released, so there's no need check for os version.